File: 2025_03/[코딩 기초 트레이닝] 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
  let oddSum = '', evenSum = '';
  num_list.forEach((num) => {
    if(num%2 === 0){
    evenSum += num
    } else {
      oddSum += num
    }
  })
  
  return +oddSum + +evenSum;
}

File: 2025_03/프로그래머스 - 코딩 기초 트레이닝/ 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
  odd_sum = ''
  even_sum = ''
  for num in num_list:
    if num % 2 == 0:
      even_sum += str(num)
    else:
      odd_sum += str(num)

  return int(odd_sum) + int(even_sum)